src/textarea.rs

```diff
@@ -1304,6 +1306,11 @@ impl<'a> TextArea<'a> {
         }
     }
 
+    /// Sets a flag indicating if the text should be masked with * characters
+    ///
+    pub fn set_password_mode(&mut self, value: bool) {
+        self.password_mode = value;
+    }
```

Comment:
Could this reasonably be an enum instead of a bool?
https://rust-lang.github.io/api-guidelines/type-safety.html#c-custom-type
Would it make sense to call this `mask` instead of `password_mode`?